src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -256,9 +255,7 @@ private IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundIsPatternExpressionOperation((BoundIsPatternExpression)boundNode);
```

Comment:
>boundCompoundAssignmentOperator.Type.IsNullableType(); [](start = 28, length = 54)

I do not think this is correct way to figure out whether the operator is lifted. Isn't this information part of the operator kind? Please check for other similar places.